### ISSUE TITLE
Ctrl+F is blocked on clean install

### ISSUE THREAD

**Issue by gulkily** [akhodakivskiy/VimFx#472]
Firefox 36.0.1
Ubuntu 14.10

1. Install VimFx on a clean profile
2. Open a website, e.g. http://www.aol.com/
3. Press Ctrl+F

Expected: Find dialog appears

Actual: Scrolls down 1 page

**Comment by lydell** [comment]
Duplicate of #308.
